proxy/tracing.go

```diff
@@ -27,6 +27,8 @@ const (
 	EndEvent           = "end"
 	StartEvent         = "start"
 	StreamHeadersEvent = "stream_Headers"
+	StreamBodyEvent    = "streamBody.byte"
```

Comment:
`stream_Body` so it's consistent with `StreamHeadersEvent`?

Reply:
I refactored this from the already used internal loop while streaming and I think it should be in-sync, wdyt?

Reply:
I can also create a different Name as you suggested for the error case

Reply:
Right, then let's keep it like this.